docs/topics/webhooks/lambda-handler.md

```diff
@@ -0,0 +1,116 @@
+-   **Topic Name**: Topic that will trigger the webhook when published.
+-   **Webhook Destination**: Endpoint URL of the webhook to POST to. _At this point, set it to a dummy value like https://none.com as it will be updated once the CloudFormation is deployed_.
+
+### Deploying the Lambda function via one-click CloudFormation
+
+Momento has taken care of the heavy lifting when it comes to building a Lambda webhook handler in addition to supplying a CloudFormation stack for deploying the resources. Initiating the deployment is as simple as clicking this button. ![CloudFormation One-Click](./cloudformation-launch-stack.png)
+
+Deploying the stack will require signing into the [AWS Console](https://console.aws.amazon.com/). Once that is completed, the CloudFormation Create Stack Screen should appear.
+
+![CloudFormation Create](./momento_stack.png)
+
+The stack will take care of the detailed parts of launching the infrastructure, but there are opportunities to customize it.
+
+-   **Stack Name**: Name of the stack when deployed.
+-   **Parameters**
+    -   _EventBridgeBus_: If using a custom bus, enter the name here. By default, the `default`` bus is used.
+    -   _MomentoSecretString_: The secret string that Momento uses to sign the payload and sends as a part of the HTTP Headers in the webhook POST. This secret string is available in the Momento Webhook Console. Copy from the below screen and paste it into this parameter
+
+![Momento Webhook Secret](./webhook_settings.jpg)
+
+### Update the Momento webhook URL
+
+Upon completion of the deployment, take the Function URL from the CloudFormation output and update the Momento webhook URL in the Momento console like the image one section above. Below is what the CloudFormation Output will look like:
+
+![CloudFormation Output](./cloudformation_output.jpg)
+
+## Verifying the Payload
+
+The ultimate destination for this webhook handler is AWS EventBridge. When working with EventBridge, Rules are the configuration point that drives what is filtered and what targets are provided the message to act upon.
+
+The Momento Lambda webhook handler puts each event from the webhook on the Event Bus with the following structure.
+
+### Example Event
+
+Below is a sample payload representing an EventBridge message as triggered from a Momento webhook. Note that this is a template and specific values will differ in your implementation.
+
+```json
+{
+    "version": "0",
+    "id": "7c7ce805-51c3-4a66-b6e0-39e4e558e6d8",
+    "detail-type": "sample-a#sample-a",
+    "source": "Momento",
+    "account": "252703795646",
+    "time": "2024-01-13T14:56:16Z",
+    "region": "us-west-2",
+    "resources": [],
+    "detail": {
+        "cache": "sample-a",
+        "topic": "sample-a",
+        "event_timestamp": 1705157775308,
+        "publish_timestamp": 1705157775308,
+        "topic_sequence_number": 2,
+        "token_id": "",
+        "text": "{\"someKey\":\"someValue\"}"
+    }
+}
+```
+
+### Event Structure
```

Comment:
Our style guide only capitalizes the first word in a heading, can you please go through and update capitalization. 
The exception to the rule is proper nouns/products and code/properties like you have below.